modules/cgrindel_bazel_starlib/metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "homepage": "https://github.com/cgrindel/bazel-starlib",
+    "maintainers": [
+        {
+            "email": "chuck.grindel@gmail.com",
+            "github": "cgrindel",
+            "name": "Chuck Grindel"
+        }
+    ],
+    "repository": [
+        "github:OWNER/REPO"
```

Comment:
@cgrindel this needs to be updated

Reply:
Bummer. I thought it was an auto replacement. I will fix and push another release. Thanks.